apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -23,9 +23,15 @@ var styles = {
 var ApplabVisualizationColumn = React.createClass({
   propTypes: {
     isReadOnlyWorkspace: React.PropTypes.bool.isRequired,
+    instructionsInTopPane: React.PropTypes.bool.isRequired,
+    visualizationHasPadding: React.PropTypes.bool.isRequired,
     hideSource: React.PropTypes.bool.isRequired,
+    isShareView: React.PropTypes.bool.isRequired,
```

Comment:
They're existing props that were in the connect, but had not been listed in the propTypes.